README.md

```diff
@@ -5,8 +5,8 @@ EntityFrameworkCore.SqlServer.HierarchyId
 
 Adds hierarchyid support to the SQL Server EF Core provider.
 
-Moved to EF Core
-----------------
+⚠️ Moved to EF Core
```

Comment:
@bricelam dot intentional?

Reply:
Rendering as expected for me (warning sign emoji)